src/styles/blocks/cardBuy.scss

```diff
@@ -0,0 +1,24 @@
+.card__buy {
+  box-sizing: border-box;
+  height: 40px;
+  width: 166px;
+  text-decoration: none;
+  text-transform: uppercase;
+  text-align: center;
+  background: $link-bg-color;
+  color: $link-text-color;
+  border-radius: 5px;
+  margin-top: 0;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 40px;
+  border: 1px solid $link-bg-color;
+
+  &:hover {
+    box-sizing: border-box;
+    background-color: $link-text-color;
+    color: $link-bg-color;
+    border: 1px solid $link-bg-color;
```

Comment:
The border property does not change on hover, so there's no need to redeclare it. If you want to change the border color on hover, you should specify only the border-color property.